fast_barcode_scanner/pubspec.yaml

```diff
@@ -13,10 +13,7 @@ dependencies:
   flutter:
     sdk: flutter
   fast_barcode_scanner_platform_interface:
-    git:
-      url: https://github.com/jhoogstraat/fast_barcode_scanner
-      ref: develop
-      path: fast_barcode_scanner_platform_interface
+    path: ../fast_barcode_scanner_platform_interface

```

Comment:
I don't get how this is handled properly...
This won't work in other project, except the example.

Reply:
This is a mistake. I set it this way during development, but it should be set to a versioned dependency. before we deploy a new version.

Reply:
@jhoogstraat , you'll have to let me know how you'd like to proceed. It seems that ideally we would merge this as-is. Plublish a new version of the platform interface, then update this pubspec to reference the new one.

Referencing a transitive git dependency can be dangerous and break builds for people. If we were to merge a breaking change to develop later on, it would break anyone who depends on this version since they would be pulling in the latest code and not a semver version.

Reply:
We should proceed the way you just outlined. 
The branch is called "develop" for a reason!